### PR TITLE
Introduce CM_Debug::log($message)

### PR DESCRIPTION
**Pull request by zazabe**

### CODE REVIEW

**Comment by zazabe** [comment]
@fvovan please review

File: library/CM/Debug.php [hunk 1]
```diff
@@ -58,4 +58,19 @@ public function setEnabled($state) {
     public static function getInstance() {
         return CM_Service_Manager::getInstance()->getDebug();
     }
+
+    /**
+     * @param {String} $message
```
Comment (fvovan) [inline library/CM/Debug.php]: please fix the docu (remove curly braces)

Comment (zazabe) [inline library/CM/Debug.php]: jsdoc!

**Comment by zazabe** [comment]
@fvovan please check again

**Comment by fvovan** [comment]
lgtm

**Pull request merged**